plugins/action/iptables_state.py

```diff
@@ -0,0 +1,189 @@
+# Copyright: (c) 2020, quidame <quidame@poivron.org>
```

Comment:
You need to rename this file to plugins/action/system/iptables_state.py, and add a relative symlink plugins/action/iptables_state.py pointing to this file.